src/status_im2/contexts/chat/home/chat_list_item/view.cljs

```diff
@@ -208,16 +208,40 @@
      {:color color
       :size  :medium}]))
 
+(defn notification
+  [{:keys [muted group-chat unviewed-messages-count unviewed-mentions-count]}]
+  (let [customization-color (rf/sub [:profile/customization-color])
+        unread-messages?    (> unviewed-messages-count 0)
+        unread-mentions?    (> unviewed-mentions-count 0)]
```

Comment:
`pos?` can be used here

Reply:
@ibrkhalil 
I'm not a big fan of using `pos?` for saying a number is greater than zero. I know it means the same but to me, using `pos?` makes me think about the value, and I start thinking what about `neg?` and also `zero?` could this value be negative?.

So, unless we always prefer to avoid `(> number 0)`, I'd prefer to keep it.

wdyt?

Reply:
Personally, pos? is way more readable to me.

Reply:
Fixed!